vip-cache-manager/vip-cache-manager.php

```diff
@@ -11,6 +11,9 @@
 require_once( __DIR__ . '/api.php' );
 
 class WPCOM_VIP_Cache_Manager {
+	const MAX_PURGE_URLS = 100;
+	const MAX_BAN_URLS   = 10;
```

Comment:
Need to decide what makes sense for these max numbers.

Reply:
This seems like a good place to start.